pagopa/cgn-onboarding-portal-frontend
# fix: [PE-859, PE-876] make portal usable at half screen size

## Short description
Added minimal fixes to make work the portal at half screen on desktop.
Visually nothing should have changed expect for smaller breakpoints.

## How to test
- open FE UAT and this PR side by side and check how screens behave per each breakpoint
